src/test/scala/com/gu/support/workers/EndToEndSpec.scala

```diff
@@ -27,7 +27,10 @@ class EndToEndSpec extends LambdaSpec {
       .chain(new CreatePaymentMethod())
       .chain(new CreateSalesforceContact())
       .chain(new CreateZuoraSubscription())
-      .parallel(new ContributionCompleted, new SendThankYouEmail())
+      .parallel(
+        new SendThankYouEmail(),
+        new SendAcquisitionEvent(MockOphanHelper.mockServices) //We have to mock Ophan - there doesn't seem to be a test mode
```

Comment:
I noticed that this wasn't included in the test, but thought it would be useful to continue testing parallel steps, even if one is just calling a mock?